app/src/main/java/org/aplatanao/billing/cockpit/control/LogTable.java

```diff
@@ -0,0 +1,32 @@
+package org.aplatanao.billing.cockpit.control;
+
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import org.aplatanao.billing.cockpit.models.Log;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LogTable extends TableView {
```

Comment:
join the constructor functionality of this class with the extending `Log` class and remove this one